lccs_db/utils.py

```diff
@@ -0,0 +1,21 @@
+#
+# This file is part of Land Cover Classification System Database Model..
+# Copyright (C) 2021 INPE.
+#
+# Land Cover Classification System Database Model. is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+
+"""Utility for Land Cover Classification System Database Model."""
+import mimetypes
+
+
+def validate_mimetypes(style_file):
```

Comment:
I think this method may have a name that refers to the type of recovery. For example, get_mimetype or something like that. What do you think?